src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -71,6 +75,81 @@ You can use headers like dynamic API keys in your function, which give you acces
 Learn more about dynamic API keys
 {% /arrow_link %}
 
+# Hot reloading {% #hot-reloading %}
+
+You can hot reload your locally developed functions, which means you can update the function code and see the changes happen in real time without restarting. Compiled languages must translate the source code into machine code, causing the system to rebuild continuously. However, interpreted languages only rebuild when you change the dependency file. Otherwise, updates can be made through a file swap.
```

Comment:
`You can hot reload your` sounds like you need to do something. Wording along likes `by default, Appwrite CLI hot-reloads your function` might be better